### PR TITLE
Working testgenerator

### PR DESCRIPTION
**Pull request by vilmakoo**
Gets test templates from the database.

(After deploying this it should be possible to add test templates straight to database.)

 :new_moon_with_face: :new_moon_with_face: :new_moon_with_face:

### CODE REVIEW

File: spec/factories.rb [hunk 1]
```diff
@@ -10,6 +10,148 @@
...
+          private void toimii(int input, String output) {
+              String inputAsString = "" + input;
+
+              ReflectionUtils.newInstanceOfClass(Submission.class);
+              io.setSysIn(inputAsString);
+              Submission.main(new String[0]);
+
+              String out = io.getSysOut();
+
+              assertTrue("Kun syöte oli '" + inputAsString.replaceAll("\\n", "\\\\\\n") + "' tulostus oli: '" + out.replaceAll("\\n", "\\\\\\n") + "', mutta se ei sisältänyt: '" + output.replaceAll("\\n", "\\\\\\n") + "'.", out.contains(output));
```
Comment (pirtnea) [inline spec/factories.rb]: These escapes are amazing :+1:

Comment (vilmakoo) [reply spec/factories.rb]: wanha

Comment (pirtnea) [reply spec/factories.rb]: Even if they are old, I still love them :cry:

Comment (vilmakoo) [reply spec/factories.rb]: Oldie but goldie.

**Pull request merged**